lib/src/linked_in_user_model.dart

```diff
@@ -68,7 +68,7 @@ class _LinkedInLocalInfo {
 
   factory _LinkedInLocalInfo.fromJson(Map<String, dynamic> json) =>
       _LinkedInLocalInfo(
-        label: json['en_US'], // possible error
+        label: json.values.toList()[0], // possible error
```

Comment:
We need a more secure way. Hardcoding can produce the same issue in this place. I will add more lines, so that we can merge it.